src/lib/components/input/input.stories.tsx

```diff
@@ -0,0 +1,135 @@
+
+import { ChangeEvent, useState } from 'react';
+import { Input, InputProps } from '.';
+import sharedConfig from './stories/config';
+
+const story = {
+  title: 'JSX/Inputs/Input',
+  component: Input,
+  argTypes: sharedConfig,
+  parameters: {
+    componentSubtitle: 'The default input component is used to gather informations from the user.'
+  }
+};
+
+export const InputStory = ({ 
+  onChange,
+  className,
+  placeholder,
+  value,
+  label,
+  hideLabel,
+  prefix,
+  error
+}: InputProps) => {
+  const [newValue, setValue] = useState(value);
+
+  const handleOnChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setValue(event.target.value)
+    onChange?.(event);
+  }
+
+  return (
+    <Input 
+      onChange={handleOnChange}
+      value={newValue}
+      className={className}
+      placeholder={placeholder}
+      label={label}
+      hideLabel={hideLabel}
+      prefix={prefix}
+      error={error}
+    />
+  );
+};
+
+InputStory.storyName = 'Input';
+
+export const InputWithPlaceholderAndNoLabel = ({ 
```

Comment:
After playing a bit with the story, I'm wondering if we really need these different stories.
In my opinion is more useful to see how the component changes using the controls. In this PR, the controls only work with the default Input and this could be confusing (for example if I click on the "Input with prefix" story I cannot edit the prefix using the controls)



What do you think?

Reply:
Yeah, that sounds good! I usually just use stories to show different examples that are worth showing (I guess it's more useful for stuff like ComparisonTable where it's more complex to show in a single place)

I'll remove these stories, there's not much value added here